lib/hexpm_web/controllers/api/user_controller.ex

```diff
@@ -53,6 +53,14 @@ defmodule HexpmWeb.API.UserController do
     end
   end
 
+  def audit_logs(conn, params = %{"name" => name}) do
```

Comment:
Do we need to add `authorize` for this endpoint?

```elixir
  plug :authorize, [domain: "api", resource: "read"] when action in [:me, :audit_logs]
```

Reply:
Yes, this information is private.

Reply:
Then I'm wondering if the body of this controller endpoint needs to be changed.

Currently, it will get the user via `Users.public_get/1` and return all audit_logs belong to this user.

Shall we change it so it only returns audit_logs for current user?

Reply:
Also, shall we use the route `GET /api/users/me/audit_logs` for this?

Reply:
That's a good point, using `GET /api/users/me/audit_logs` for this makes sense.

Reply:
`GET /api/users/:name/audit_logs` was removed in 6a1b5e38

`GET /api/users/me/audit_logs` was added in 5eae3266